features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2843,4 +2843,12 @@
     <AllowCrossTenantTokenIntrospection>{{oauth.allow_cross_tenant_token_introspection}}</AllowCrossTenantTokenIntrospection>
 
 
+    <!-- Claims which must not delete during the syncing process of existing claim mappings with IDP claim mappings
+    for JIT provisioned user. -->
+    <IndelibleClaims>
```

Comment:
this config is used only during JIT provisioning right?
if that's the case let's move it under `<JITProvisioning>` element.

Reply:
Addressed with commit 741cc49f237977e51f19977a9e9d3da6ea485e18